packages/opencensus-web-instrumentation-zone/src/zone-types.ts

```diff
@@ -31,6 +31,7 @@ export type AsyncTask = Task & {
 
 /** Data used to create a new OnPageInteractionStopwatch. */
 export interface OnPageInteractionData {
+  originLocation: string;
```

Comment:
What would you think about calling this `startLocationHref` or similar? When I hear "origin" I think of the origin (host) of the site, and I think adding the `Href` at the end is nice in case you add things like path and hash too.